nipype/interfaces/fsl/ICA_AROMA.py

```diff
@@ -0,0 +1,115 @@
+#ICA_AROMA pulled from: https://github.com/rhr-pruim/ICA-AROMA
+#This assumes ICA_AROMA.py is already installed and callable via $PATH
+from nipype.interfaces.base import (
+    TraitedSpec,
+    CommandLineInputSpec,
+    CommandLine,
+    File,
+    Directory,
+    traits,
+    OutputMultiPath
+)
+import os
+
+class ICA_AROMAInputSpec(CommandLineInputSpec):
+    featDir = Directory(exists=True,
```

Comment:
I haven't heard of flake8, that looks really useful, thanks!

argstr: D'oh, I made a couple of errors where I didn't specify the string correctly

I'll move 'desc' to the end and move 'mandatory' next to 'exists'. Question: if mandatory=False is the default behavior, should I still be explicit or  is that too wordy?